library/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="emoji_background">#ECEFF1</color>
+    <color name="emoji_icons">#61000000</color>
```

Comment:
I calculated that from the [material guidelines](https://material.io/guidelines/style/color.html#color-text-background-colors) for icons. The suggestion there is `#000000` with a opacity of 54%, which correspondents to 96,9 (54% of the max value 255) = roughly 61 as hex.